Solutions/Endjin.SpecFlow.Selenium.Framework.Net45/Environment/TestEnvironment.cs

```diff
@@ -44,7 +44,12 @@ static TestEnvironment()
 
         private TestEnvironment()
         {
-            var config = new FileInfo(TestContext.CurrentContext.TestDirectory + @"\WebsiteUnderTest\web_test.config.json");
+            FileInfo config = new FileInfo(@"WebsiteUnderTest\web_test.config.json");
+            if (!config.Exists)
```

Comment:
This is one of those classic race condition problems with file manipulation. It is quite possible that the file will stop existing between that test and you then going and reading everything. Is the intent of this that you are looking for the file in the current location, or in the currently set test directory? Why can't you just use the second case all the time - is it something to do with running in a different environment. This is the kind of code that needs some comments to explain what is going on to maintainers.